### PR TITLE
Finalize handling of MP tensors

### PR DESCRIPTION
**Pull request by atobiszei**
* Fix some compiler warnings

CVS-116277

### CODE REVIEW

**Review by rasapala — APPROVED** [review]
LGTM

File: mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc [hunk 1]
```diff
@@ -581,16 +574,23 @@ class ModelAPISideFeedCalculator : public CalculatorBase {
                 LOG(INFO) << "OVMS calculator will process vector<TfLiteTensor>";
                 auto outputStreamTensors = std::vector<TfLiteTensor>();
                 if (!this->initialized) {
-                    interpreter_->AddTensors(output.size()); // HARDCODE
-                    interpreter_->SetInputs({0,1}); // HARDCODE was 0 for single input
+                    interpreter_->AddTensors(output.size());
+                    std::vector<int> indexes(output.size());
+                    std::iota(indexes.begin(), indexes.end(), 0);
+                    interpreter_->SetInputs(indexes);
+                    RET_CHECK(output.size() == 2);
```
Comment (dkalinowski) [inline mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc]: Do hardcode 2 outputs here?

File: mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc [hunk 2]
```diff
@@ -581,16 +574,23 @@ class ModelAPISideFeedCalculator : public CalculatorBase {
...
                     for (auto& [name,tensor] : output) {
                         std::vector<int> tfliteshape;
                         for (auto& d : tensor.get_shape()) {
                             tfliteshape.emplace_back(d);
                         }
-                        interpreter_->SetTensorParametersReadWrite(/*tensor_index*/tensorId, kTfLiteFloat32, name.c_str(),
-                                                       tfliteshape, TfLiteQuantization());
+                        interpreter_->SetTensorParametersReadWrite(
+                                        /*tensor_index*/tensorId,
+                                        kTfLiteFloat32, // TODO datatype
```
Comment (dkalinowski) [inline mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc:593]: Do we support other precisions than float32 in this release?

Comment (atobiszei) [reply mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc:593]: Not for TfLite

File: mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc (hunk 1 above)
Comment (atobiszei) [reply mediapipe/calculators/ovms/modelapiovmsinferencecalculator.cc]: removed check

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**